src/main/java/org/jenkinsci/plugins/gitclient/verifier/KnownHostsFileVerifier.java

```diff
@@ -54,4 +64,11 @@ public boolean verifyServerHostKey(String hostname, int port, String serverHostK
         }
     }
 
+    private void logHint(TaskListener listener) {
+        listener.getLogger().println(HyperlinkNote.encodeTo("https://plugins.jenkins.io/git-client/#plugin-content-ssh-host-key-verification","You're using 'Known host file' strategy to verify ssh host keys," +
```

Comment:
Usually phrased as plural
```suggestion
        listener.getLogger().println(HyperlinkNote.encodeTo("https://plugins.jenkins.io/git-client/#plugin-content-ssh-host-key-verification","You're using 'Known hosts file' strategy to verify ssh host keys," +
```